### PR TITLE
BaSD-MS-58: Final form positions

### PR DESCRIPTION
**Pull request by MicaQuarleri**
The controller file was change in order to send an object and not onlya msg

### CODE REVIEW

File: src/Components/Admin/Positions/Form/index.js [hunk 1]
```diff
@@ -3,121 +3,66 @@ import styles from './form.module.css';
...
-  }, [selectedPosition]);
+    dispatch(getClients());
+    if (positionId) {
+      dispatch(getOnePosition(positionId));
+    } else {
+      dispatch(selectedToDefault());
+    }
+  }, []);
+
+  console.log(selectedPosition);
```
Comment (ignacioaristo) [inline src/Components/Admin/Positions/Form/index.js:39]: Remove this `console.log`

**Review by ignacioaristo — CHANGES_REQUESTED** [review]
I couldn't find the error, but if you Create a new `positions `the app crash. When you Update a `position `the app works perfectly

**Review by leimeterjoaquin — APPROVED** [review]
Nice, remove console log before merging!

**Review by ignacioaristo — APPROVED** [review]
Perfect

**Pull request merged**